frontend/src/metabase/public/components/widgets/EmbedModalContent.unit.spec.tsx

```diff
@@ -137,25 +141,15 @@ describe("EmbedModalContent", () => {
   });
 });
 
-const storeInitialState = {
-  currentUser: {
-    is_superuser: true,
-  },
-  settings: {
-    values: {
-      "enable-embedding": true,
-      "embedding-secret-key": "my_super_secret_key",
-    },
-  },
-};
-
 function renderWithConfiguredProviders(element: JSX.Element) {
   renderWithProviders(element, {
-    storeInitialState,
-    reducers: {
-      settings: (state: Record<string, any> = storeInitialState.settings) => {
-        return state;
-      },
+    storeInitialState: {
+      settings: createMockSettingsState({
```

Comment:
based on how `settings` look like:
https://github.com/metabase/metabase/blob/c92e2796672619428c7379d490372a9ecb3810b0/frontend/src/metabase-types/store/settings.ts#L3-L5

Seems the only `values` is valid there. What do you think about making creating a mock store for settings easier like maybe we could assume anything inside `values` would be passed to `createMockSettings` under the hood. Otherwise, having to import both `createMocoksetttingsState` and`createMockSettings` feel too cumbersome and is contrary to the goal of the epic itself [Improve frontend unit-testing experience](https://github.com/metabase/metabase/issues/26960#top)
#

I'm thinking maybe the `createMockSettingsState` would be simplified to something like:

```ts
createMockSettingsState({
  values: {
    'enable-embedding': true,
    'embedding-secret-key': 'my_super_secret_key',
  },
})
```
or even
```ts
createMockSettingsState({
  'enable-embedding': true,
  'embedding-secret-key': 'my_super_secret_key',
})
```
What do you think?

Reply:
Yeah, agreeing that's the right thing to do 👍 Let's fix this in another PR though so that one doesn't become too big

Reply:
sounds good to me 👍